mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -295,6 +295,20 @@ def test_pg_settings_dialog(raw_orig, pg_backend):
     )
     assert inv_norm == ch_spinbox.value()
 
+    ch_scale_spinbox = fig.mne.fig_settings.ch_scaling_spinboxes[ch_type_test]
+    ch_sens_spinbox = fig.mne.fig_settings.ch_sensitivity_spinboxes[ch_type_test]
+    scaling_spinbox_value = ch_spinbox.value()
+    sensitivity_spinbox_value = ch_sens_spinbox.value()
+    scaling_value = fig.mne.scalings[ch_type_test]
+    new_scaling_spinbox_value = scaling_spinbox_value * 2
+    new_expected_sensitivity_spinbox_value = sensitivity_spinbox_value * 2
+    ch_scale_spinbox.setValue(new_scaling_spinbox_value)
+    new_scaling_value = fig.mne.scalings[ch_type_test]
+    assert scaling_value != new_scaling_value
+    np.testing.assert_allclose(
+        ch_sens_spinbox.value(), new_expected_sensitivity_spinbox_value, atol=2
```

Comment:
Why 2? It seems like `atol=0.1` (or maybe 0.2) should be enough?

Reply:
... let's see if CIs agree
```suggestion
        ch_sens_spinbox.value(), new_expected_sensitivity_spinbox_value, atol=0.1
```